bokehjs/test/integration/regressions.ts

```diff
@@ -4345,4 +4345,39 @@ describe("Bug", () => {
       await view.ready
     })
   })
+
+  describe("in issue #14207", () => {
+    it("has zoom in when visibility changes", async () => {
```

Comment:
The error:
```
 ✗ Bug
 └┬─ in issue #14207
  └── has zoom in when visibility changes
images differ by 1px (0.00%)
```
can be resolved with:
```suggestion
    it.allowing(1)("has zoom in when visibility changes", async () => {
```
This is typically due to CSS shadows. Or you can replace select with something simpler, e.g. a `Spacer` or a `Pane` with a little bit of CSS for width, height and maybe background color.